tools/topic_tools/package.xml

```diff
@@ -28,6 +28,8 @@
   <build_depend>xmlrpcpp</build_depend>
 
   <run_depend>message_runtime</run_depend>
+  <run_depend>python-numpy</run_depend>
+  <run_depend>python-texttable</run_depend>
```

Comment:
Sure, but adding more and more features into ros_comm also means offloading the maintenance work to me :wink: